docker-compose-cloud.buildx.yaml

```diff
@@ -100,3 +100,19 @@ services:
         platforms:
           - linux/amd64
           - linux/arm64
+  connector-builder:
```

Comment:
Does this mean we are planning to add the connector builder server to cloud as well? That would be great if we are able to do that somewhat easily for the first iteration, since then users can just go to airbyte cloud to access the connector builder (we could even deploy it to demo.airbyte.io so that they don't even need to log in to try it out)

Reply:
removed from the PR as this is out of scope. Happy to revisit after the initial release